features/membership_status/member-status-change.feature

```diff
@@ -0,0 +1,101 @@
+Feature: Membership status updated due to payments or expiration
+
+  As a member
+  So that my membership status is always current and accurate
+  The system must review dates, payments, and actions nightly
+
+  Membership application state, membership payments, and H-branding (license) payments
+  all affect membership status.
+  The particular date also affects status:  has a membership expired?
+  These all need to accurately change membership status.
+
+  Background:
+    Given the following users exist
+      | email               | admin | member | membership_number |
+      | emma@mutts.com      |       | true   | 1001              |
+      | bob@snarkybarky.com |       | true   | 1002              |
+      | lars@newapp.com     |       | false  |                   |
+      | admin@shf.se        | true  | false  |                   |
+
+    Given the following companies exist:
+      | name                 | company_number | email                  | region    |
+      | HappyMutts           | 2120000142     | woof@happymutts.com    | Stockholm |
+      | No More Snarky Barky | 5560360793     | snarky@snarkybarky.com | Stockholm |
+      | LarsCo               | 5562252998     | lars@newapp.com        | Stockholm |
+
+    Given the following applications exist:
+      | user_email          | company_number | category_name | state    |
+      | emma@mutts.com      | 2120000142     | rehab         | accepted |
+      | bob@snarkybarky.com | 5560360793     | grooming      | accepted |
+      | lars@newapp.com     | 5562252998     | rehab         | new      |
+
+
+    Given the following payments exist
+      | user_email          | start_date | expire_date | payment_type | status | hips_id | company_number |
+      | emma@mutts.com      | 2017-12-31 | 2018-12-31  | member_fee   | betald | none    | 2120000142     |
+      | emma@mutts.com      | 2017-12-31 | 2018-12-31  | branding_fee | betald | none    | 2120000142     |
+      | bob@snarkybarky.com | 2018-03-01 | 2019-02-28  | member_fee   | betald | none    | 5560360793     |
+      | bob@snarkybarky.com | 2018-03-02 | 2019-03-01  | branding_fee | betald | none    | 5560360793     |
+
+
+
+  # TODO should these be put into already existing .feature files?
+
+  # --- MEMBERSHIP PAYMENTS -------------
+
+  @time_adjust
+  Scenario: Membership payment made before membership expires
```

Comment:
After merging with the current development, I'm having trouble getting this to pass.

Does it pass for you?

(and after #562 is merged, this will change)